### ISSUE TITLE
Add a toggle to recipe view to show the full required amount per ingredient

### ISSUE THREAD

**Issue by philip-n** [TandoorRecipes/recipes#1325]
### Is your feature request related to a problem? Please describe.

Before starting to cook a recipe, I often collect all ingredients (also serves as a quick check that everything is available).

Now when a recipe is split into multiple steps, the ingredients overview at the top is also split accordingly. As ingredients may occur in multiple steps, there is no possibility to see at one glance how many ingredients are required.

E.g. first step needs one egg, second step none, third step two eggs. I need to "manually" parse it.

### Describe the solution you'd like

Offer a toggle that switches between the current view (ingredients per step) and a "merged" view (all ingredients of the recipe, summed up over all sub-steps).

I guess this is more or less the functionality that is already available in the shopping list (?).

### Describe alternatives you've considered

Doing it by hand or adding the recipe to a shopping list and looking the quantities up there.

### Contribute

- [ ] Yes
- [ ] Partly
- [X] No

**Comment by philip-n** [comment]
Very loosely related to https://github.com/TandoorRecipes/recipes/issues/1293 .

**Comment by vabene1111** [comment]
I see how this might be useful, I guess it should be implemented together with the other issue.

**Comment by xMAC94x** [comment]
+1 for summed up overview.
IMO this toggle should be default. There is no advantage of a Ingredients overview if it just shows the exact copy from whats below in the single steps. If you want to see each single values per step you can just scroll down

**Comment by vabene1111** [comment]
there are certainly advantages and those have been discussed before but i agree, this will be implemented.

**Comment by brhahlen** [comment]
Agreed, this would be a very useful addition, especially for recipes where you need to use different amounts of an ingredient over multiple steps

**Comment by brhahlen** [comment]
@vabene1111 Would it be possible to include this in 1.4.0, by any chance? 
I think this is one of the most lacking features in an otherwise great application!

**Comment by stuartf** [comment]
I'd like to suggest that the ingredient summary should also note the combined ingredients are `divided`. i.e. step 1 uses 1 tbsp butter and step 3 uses 2 tbsp butter, recipe summary says "3 tbsp butter (divided)"

**Comment by mondeg0** [comment]
This feature is really missing in this amazing application, and the work is already partially done in the shopping list... I would use the same grouping as done there and just keep the three relevant fields (amount unit name).

Currently, the behavior is unclear and duplicates information found in the steps. To enhance clarity, I recommend adding a toggle option for a concise view that displays only the ingredients in the steps, eliminating the text from there and keeping the overview as a real overview of the total ingredients needed.

For recipes with many steps, like a mousse cake, where several steps are required with a good amount of repeating ingredients, the overview is confusing.

**Comment by SnoFox** [comment]
Hi team,

Has anyone come up with a reasonable workaround for this missing feature? I'm considering calculating the overview by hand for the default serving size and adding it as the first step in the recipe, and using Tampermonkey/Adblock to disable the overview. I have a use-case that fits squarely in both #1293 as well as this one, #1325.

I split my recipes both into sub-recipes that I can prep in bulk and store and very small steps for the day of (ex, 1. Prep produce, 2. Make sauce, 3. Cook meat), so I end up with overviews that are effectively useless for validating I have all the ingredients out.

<img width="542" alt="image" src="https://github.com/user-attachments/assets/6b2677df-f7a6-4647-a4c2-41afee73f5ef">

<img width="558" alt="image" src="https://github.com/user-attachments/assets/e3719f34-4d13-4af2-b536-e874fdc68891">

**Comment by Aidan-OS** [comment]
Hopefully this will have a chance to get added soon now that the V2 upgrade has released. It's the one thing holding Tandoor back from being absolutely perfect for me.

**Comment by vabene1111** [comment]
took only 3 years but now its implemented :)
